tests/test_metadata.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import os
+
+from wkcuber.cubing import ensure_wkw
+from wkcuber.utils import WkwDatasetInfo, open_wkw
+from wkcuber.metadata import (
+    write_webknossos_metadata,
+    refresh_metadata,
+    read_datasource_properties,
+    read_metadata_for_layer,
+)
+
+
+def test_element_class_convertion():
+    test_wkw_path = os.path.join("testoutput", "test_metadata")
+    prediction_layer_name = "prediction"
+    prediction_wkw_info = WkwDatasetInfo(
+        test_wkw_path, prediction_layer_name, np.float32, 1
```

Comment:
Prediction layers usually have three channels which you could reflect here.

Reply:
The `1` is the `Mag` and the `WKWDatasetInfo` does not contain a `num_channel`-variable.